site.step-15-varnish.yml

```
@@ -76,6 +76,10 @@
       varnish_debug_request_query_param_name: "{{ mageops_debug_token_query_param }}"
       varnish_debug_request_cookie_name: "{{ mageops_debug_token_request_cookie }}"
       varnish_debug_request_header_name: "{{ mageops_debug_token_http_header }}"
+      varnish_qa_request_token: "{{ mageops_debug_token }}"
```

shouldn't it be `mageops_qa_token`?


```suggestion
      varnish_qa_request_token: "{{ mageops_qa_token }}"
```